plot-test/plot/tests/PRs/90.rkt

```diff
@@ -0,0 +1,81 @@
+            (test-suite "PR90: plot-metrics plane-vector"
+                        (check-equal? (send pm plane-vector)
+                                      #(0 0 1)))))
+    
+    (let ([pmbm (plot3d-bitmap (polar3d (λ (ϕ θ) ϕ)))])
+      (list (test-suite "PR90: bitmap" (check-true (is-a? pmbm bitmap%)))
+            (test-suite "PR90: 3d/bitmap get-plot-bounds"
+                        (check-within (send pmbm get-plot-bounds)
+                                      #(#(-3.287703432872698 6.282003882664296)
+                                        #(-4.812750657123522 1.819088586099858)
+                                        #(-6.283185307179587 6.283185307179587))
+                                      1e-15))
+            (test-suite "PR90: 3d/bitmap plot->dc & dc->plot"
+                        (check-within (send pmbm plot->dc (send pmbm dc->plot #(200 200)))
+                                      #(200 200) 1e-15))
+            (test-suite "PR90: 3d/bitmap plane-vector"
+                        (check-equal? (send pmbm plot->dc (send pmbm plane-vector))
+                                      (send pmbm plot->dc #(0 0 0))))))
+    
+    (let ([pp (plot-pict (function (λ (x) x) 1 2))])
+      (list (test-suite "PR90: plot-pict" (check-true (and (plot-pict? pp) (pict? pp))))
+            (test-suite "PR90: plot-pict get-plot-bounds"
+                        (check-equal? (plot-pict-bounds pp)
+                                      #(#(1 2) #(1 2))))
+            (test-suite "PR90: plot-pict plot->dc & dc->plot"
+                        (check-within ((plot-pict-dc->plot pp) ((plot-pict-plot->dc pp) #(0 0)))
+                                      #(0 0) 1e-15))
+            (test-suite "PR90: plot-pict plane-vector"
+                        (check-equal? (plot-pict-plane-vector pp)
+                                      #(0 0 1)))))
+
+    (let* ([ps (plot3d-snip (surface3d (λ (x y) (+ x y)) 0 1 1 2))]
+           [plotcoords #(1 1 1)]
+           [coords (send ps plot->dc plotcoords)])
+      (list (test-suite "PR90: 3d/snip" (check-true (is-a? ps plot-metrics<%>)))
+            (test-suite "PR90: 3d/snip get-plot-bounds"
+                        (check-equal? (send ps get-plot-bounds)
+                                      #(#(0 1) #(1 2) #(1 3))))
+            (test-suite "PR90: 3d/snip plot->dc & dc->plot"
+                        (check-within (send ps plot->dc (send ps dc->plot #(200 200)))
+                                      #(200 200) 1e-15))
+            (test-suite "PR90: 3d/snip plane-vector"
+                        (check-equal? (send ps plot->dc (send ps plane-vector))
+                                      (send ps plot->dc #(0 0 0))))
+            (test-suite "PR90: 3d/snip before resize"
+                        (check-equal? (send ps plot->dc plotcoords) coords))
+            (test-suite "PR90: 3d/snip after resize"
+                        (send ps resize 800 800)
+                        (sleep/yield .5)
```

Comment:
Unfortunately, this sleep call will create a flaky test which might fail depending on what machine it is run on and the overall CPU load on that machine.  Unfortunately, I don't have a suggestion on how to carry out this test, so I think this should be disabled for now. 

I think the testing is sufficient for now.

Reply:
Will it be ok if I leave the test in as a comment?

Reply:
Yes, it should be left in as a comment -- it would be a good idea to be able to test these things, but I don't know how.

Also, you will need to merge the master branch into this one.

Reply:
There ought to be something that can wait for the resize to complete -- maybe @rfindler knows.